components/webdriver_server/actions.rs

```diff
@@ -484,4 +498,159 @@ impl Handler {
             thread::sleep(Duration::from_millis(POINTERMOVE_INTERVAL));
         }
     }
+
+    // https://w3c.github.io/webdriver/#dfn-dispatch-a-scroll-action
```

Comment:
```suggestion
    /// https://w3c.github.io/webdriver/#dfn-dispatch-a-scroll-action
```
This way it becomes a Rust documentation comment